grails-app/services/org/bbop/apollo/SequenceService.groovy

```diff
@@ -284,6 +318,55 @@ class SequenceService {
         }
     }
 
+    def loadGenomeFasta(Organism organism, JSONObject referenceTrackObject) {
+        organism.valid = false;
+        organism.save(flush: true, failOnError: true, insert: false)
+
+        String genomeFastaFileName = organism.directory + File.separator + referenceTrackObject.urlTemplate
+        String genomeFastaIndexFileName = organism.directory + File.separator + referenceTrackObject.faiUrlTemplate
```

Comment:
Can we use a default of $genomeFastaFileName.fai  if not provided?

Reply:
Why is that? If we are reading from the trackList.json then both `urlTemplate` and `faiUrlTemplate` should be defined.

Reply:
They don't have to, just more thinking that if it 90% of the time it follows this pattern, it would be a reasonable default.   Not a big deal either way.   Now that I think about it, it probably breaks somewhere else if not defined there.

Reply:
Though, wondering if the trackList.json could be modified on reading to automatically add the default AND specify the `"useAsRefSeqStore": true ` if not present.

Reply:
Since the data generation pipeline always uses prepare-refseqs.pl script, the data directory will definitely have a trackList.json, a seq folder, the fasta genome and the fasta index.

We can make this more obvious in our documentation too.

Reply:
I think that longer-term we want to not always have them use prepare-refseqs.pl BUT they will probably use another tool, so you are right.   Yeah, just a little documentation then.

Reply:
Right. Because it would be tough to ask people to generate the `refSeqs.json` which is important from a JBrowse perspective.
In the future, we might do away with the whole `seq` folder hierarchy and simply have a FASTA file, its index and a `trackList.json`, which would be more neater.